tests/unit/type/stream.tcl

```diff
@@ -102,6 +102,11 @@ start_server {
         }
     }
 
+    test {XADD with NOMKSTREAM option} {
+        r DEL mystream
+        assert_equal "" [r XADD mystream NOMKSTREAM * item 1 value a]
```

Comment:
Can you also test that XADD with NOMKSTREAM with a stream still works?

Reply:
thanks @madolson for the review, I added the test for stream exist scenario, please take look this, thanks

Reply:
let's also assert that the key doesn't exist (not just validate the command response)

Reply:
thanks @oranagra , I updated the test case to check the key doesn't exist. please review, thanks